### PR TITLE
Fix a bug that prevents an event at the recruitment of Kyobaine.

### PR DESCRIPTION
**Pull request by oo13**
Strangely enough, WML seems to decide true is not equal to a variable set to true.

### CODE REVIEW

**Review by cooljeanius — APPROVED** [review]
I haven't tested this myself, but I'll assume that you have and approve... let's get a second pair of eyes from @nemaara first before merging, though.

**Comment by Toranks** [comment]
For future information, true and false without quotes are treated as boolean status. In the case that you want to put true or false as words for some reason, you should put in "quotes".

**Comment by cooljeanius** [comment]
> For future information, true and false without quotes are treated as boolean status. In the case that you want to put true or false as words for some reason, you should put in "quotes".

...ok, so I'll take this as a review that the proposed fix is correct, and merge it.

**Pull request merged**
